pkg/core/config/schema.go

```diff
@@ -24,22 +24,33 @@ const (
 		"$id": "#TenantConfiguration",
 		"type": "object",
 		"properties": {
-			"version": { "const": "1" },
+			"version": { "const": "2" },
 			"app_id": { "$ref": "#NonEmptyString" },
 			"app_name": { "$ref": "#NonEmptyString" },
-			"app_config": {
-				"type": "object",
-				"properties": {
-					"database_url": { "$ref": "#NonEmptyString" },
-					"database_schema": { "$ref": "#NonEmptyString" }
-				}
-			},
-			"user_config": { "$ref": "#UserConfiguration" }
+			"database_config": { "$ref": "#DatabaseConfiguration" },
+			"hook": { "$ref": "#HookTimeoutConfiguration" },
+			"app_config": { "$ref": "#AppConfiguration" }
 		},
-		"required": ["version", "app_id", "app_name", "app_config", "user_config"]
+		"required": ["version", "app_id", "app_name", "database_config", "app_config"]
+	},
+	"DatabaseConfiguration": {
+		"$id": "#DatabaseConfiguration",
+		"type": "object",
+		"properties": {
+			"database_url": { "$ref": "#NonEmptyString" },
+			"database_schema": { "$ref": "#NonEmptyString" }
+		}
+	},
+	"HookTimeoutConfiguration": {
```

Comment:
I think it should not be named `HookTimeoutConfiguration`, since it may store other hook related configuration in future.

Reply:
Can you suggest a good name? HookConfiguration is already taken :(